documentation.html

```diff
@@ -122,8 +122,163 @@ <h1 class="heading--1-medium">Code</h1>
 						<!-- 	style="border: 0; transform: scale(1); overflow: hidden" -->
 					</div>
 				</section>
+				<!-- --------------------------- -->
+				<div class="u-margin-bottom-medium"></div>
+				<hr />
+				<div class="u-margin-bottom-medium"></div>
 
-				<!-- -----------ALERTS--------- -->
+				<!-- -----------BUTTONS--------- -->
+				<section class="doc-main-item" id="buttons">
+					<div class="main__header">
+						<h1 class="heading--1">Buttons</h1>
+						<h1 class="heading--2">
+							The Button component is used to trigger an action or event, such
+							as submitting a form, opening a dialog, canceling an action, or
+							performing a delete operation.
+						</h1>
+					</div>
+					<div class="main__about">
+						<h1 class="heading--1-medium">Variants</h1>
+						<h1 class="heading--3">
+							<ul class="feature__list">
+								<li class="types">
+									<span class="color-primary">
+										<code class="code-classes">btn-primary</code> :</span
+									>
+									Solid Button
+								</li>
+
+								<li class="types">
+									<span class="color-primary"
+										><code class="code-classes">btn-secondary</code> :</span
+									>
+									Button with an outline
+								</li>
+
+								<li class="types">
+									<span class="color-primary"
+										><code class="code-classes">btn-ghost</code> :</span
+									>
+									Button without an outline
+								</li>
+
+								<li class="types">
+									<span class="color-primary">
+										<code class="code-classes"> btn-link</code> :</span
+									>
+									Button for links
+								</li>
+							</ul>
+						</h1>
+					</div>
+					<div class="main__example row-container">
+						<a href="" class="btn btn-primary">Button</a>
+						<a href="" class="btn btn-secondary">Button</a>
+						<a href="" class="btn btn-ghost">Button</a>
+						<a href="" class="btn btn-link">Button</a>
+					</div>
+					<h1 class="heading--1-medium">Code</h1>
+					<div class="main__code">
+						<code class="main__code-bg">
+							<iframe
```

Comment:
If possible, avoid iframe. The reason is Iframes cause SEO problems, you can google the alternatives and can read more about them. Please correct me if I said something wrong.

Reply:
I am using carbon for the code section and it gives the snippet for iframe itself> this is why I have to use the iframe.

Reply:
Oh ok. Alright then.